manifest/ocischema/builder.go

```diff
@@ -96,8 +96,11 @@ func (mb *Builder) Build(ctx context.Context) (distribution.Manifest, error) {
 }
 
 // AppendReference adds a reference to the current ManifestBuilder.
-func (mb *Builder) AppendReference(d distribution.Describable) error {
-	mb.layers = append(mb.layers, d.Descriptor())
+//
+// The reference must be either a [distribution.Descriptor] or a
+// [distribution.Describable].
```

Comment:
```suggestion
```
The function signature has the `ref` parameter as a `Descriptor`, which is a concrete type. How could a `Describable` interface value be passed in without conversion?

Reply:
Ah, my bad; I did a rebase and looks like I had the second commit in my local fork, which I thought was finished, but didn't revert the GoDoc changes from the first one.